client/state/jitm/reducer.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import {
+	JITM_SET,
+} from 'state/action-types';
+import { combineReducers, keyedReducer } from 'state/utils';
+
+export const storeJITM = ( state = {}, { type, jitms } ) =>
+	type === JITM_SET
+		? jitms
+		: state;
+
+const sitePathJITM = keyedReducer( 'keyedPath', storeJITM );
```

Comment:
`keyedPath` seems a bit murky when working backwards to figure out what it is. would it be more clear if we used something in the name to indicate it? I'm not sure what it's supposed to look like so I don't really know.

additionally, what would be 💯 is adding an example or two of one such `keyedPath` in a comment above this line so that we might know what to expect

Reply:
💯